src/main/kotlin/kweb/KwebConfiguration.kt

```diff
@@ -0,0 +1,55 @@
+package kweb
+
+import mu.KotlinLogging
+import java.time.Duration
+
+/**
+ * A central configuration class for Kweb parameterization
+ *
+ * Please note this is not [Kweb.Feature.Configuration], which is a Ktor specific config block
+ *
+ */
+object KwebConfiguration {
```

Comment:
We can actually make this internal, but not sure if we want to - I think it's useful for folks to debug their app?

Reply:
Should this be a class such that an instance of it can be passed to the Kweb constructor, rather than being global/static?  That way, if we have multiple Kweb instances in the same project they won't all necessarily need the same configuration?

Reply:
I came up with a design on how to support this while providing some good defaults, see 2be22e6138ddf9bb8338f8bc569623a3d1309cb0

Reply:
Yes, that makes sense.